Tesis Doctorals en Xarxa.js

```diff
@@ -0,0 +1,277 @@
+
+{
+	"translatorID": "dd2d7c63-0c4b-4da4-b1d8-e72c7445c1e0",
+	"label": "Tesis Doctorals en Xarxa",
+	"creator": "CSUC",
+	"target": "^https?://(www\.)?(tdx\.cat|tesisenred\.net)",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsb",
+	"lastUpdated": "2016-09-22 15:33:33"
+}
+
+
```

Comment:
Yeah, I think what is currently missing is a `README.md` and a file `CONTRIBUTING` (which we can link from the `README.md`). A style guide could be of this? We can also discuss this further in a new issue, if you would like.